src/Reflex/Class.hs

```diff
@@ -703,6 +718,19 @@ distributeListOverDyn = distributeListOverDynWith id
 distributeListOverDynWith :: Reflex t => ([a] -> b) -> [Dynamic t a] -> Dynamic t b
 distributeListOverDynWith f = fmap (f . map (\(Const2 _ :=> Identity v) -> v) . DMap.toList) . distributeDMapOverDynPure . DMap.fromList . map (\(k, v) -> Const2 k :=> v) . zip [0 :: Int ..]
 
+-- | Create a new 'Event' that occurs when the first supplied 'Event' occurs
+-- unless the second supplied 'Event' occurs simultaneously.
+unlessE :: Reflex t => Event t a -> Event t b -> Event t a
+unlessE = alignWithMaybe $ \case { This a -> Just a ; _ -> Nothing }
```

Comment:
I know i used bracketed notation when we were discussing it before, but just for consistency with the rest of the code, let's use a new line for each match in the case statement.